ui/apps/platform/src/Containers/Clusters/Components/Scanner/ScannerStatusTotals.tsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import { healthStatusLabels } from 'messages/common';
+import { healthStatusStyles } from '../../cluster.helpers';
+
+const trClassName = 'align-bottom leading-normal'; // align-bottom in case heading text wraps
+const thClassName = 'font-600 pl-0 pr-1 py-0 text-left';
+const tdClassName = 'p-0 text-right';
+const tdErrorsClassName = 'font-600 pb-0 pl-0 pr-1 pt-2 text-left'; // pt for gap above errors
+
+type ScannerStatusTotalsProps = {
+    scannerHealthInfo: {
+        totalDesiredAnalyzerPods: number;
+        totalReadyAnalyzerPods: number;
+        totalDesiredDbPods: number;
+        totalReadyDbPods: number;
+        statusErrors: string[];
+    };
+};
+
+const notAvailable = 'n/a';
+
+const resolveDbHealthStatus = (desiredPods: number, readyPods: number) => {
+    if (!desiredPods) {
+        return notAvailable;
+    }
+    return healthStatusLabels[readyPods ? 'HEALTHY' : 'UNHEALTHY'];
+};
+
+const ScannerStatusTotals = ({ scannerHealthInfo }: ScannerStatusTotalsProps) => {
+    const {
+        totalDesiredAnalyzerPods,
+        totalReadyAnalyzerPods,
+        totalDesiredDbPods,
+        totalReadyDbPods,
+        statusErrors,
+    } = scannerHealthInfo;
+
+    return (
+        <table data-testid="scannerHealthInfo">
+            <tbody>
+                <tr className={trClassName} key="totalReadyPods">
+                    <th className={thClassName} scope="row">
+                        Scanner pods ready:
```

Comment:
Suggestion: we can reduce verbosity and make things look a little nicer if we say `Pods ready:` instead of `Scanner pods ready:`, `Version:` instead of `Collector version:` and so on. What do you think?
It means you'd need to change titles for Collector and Admission Control but that shouldn't be hard.

Compare _before_
![image](https://user-images.githubusercontent.com/537715/166205798-07c8af90-7b06-4ddc-b876-c2d0518ff2f1.png)

and _after_
![image](https://user-images.githubusercontent.com/537715/166205756-b89ff89f-b1c6-46eb-a006-4ce4382b131b.png)

Reply:
Good idea. I was thinking about simplifying even more, like: "Replicas: 3/3". In any case, it seems that this is out of the scope of this PR.